tests/conftest.py

```diff
@@ -0,0 +1,158 @@
+import pytest
+
+
+@pytest.fixture
+def transaction_type_0():
```

Comment:
Maybe more descriptive names than `0`, '1' and so on?

Reply:
the types itself are actually represented by integers and the names itself can be quite long (eg. `multi_signature_registration`) that's the only reason I went with integers. What do you think?

Reply:
good point. Leave it as it is. Maybe add a docstring if you feel it's needed. Usually it's enough just to look at the thing it does.

Reply:
I agree, will add comments to each fixture to explain what it does